stryker.conf.json

```diff
@@ -10,5 +10,9 @@
   ],
   "testRunner": "command",
   "transpilers": [],
+  "dashboard": {
+    "project": "github.com/brodybits/xmldom",
+    "version": "master"
+  },
```

Comment:
I think such a workaround implies that your master is your fork is not running that dashboard as well?
And I'm not sure if there is any implication of your repo not being completely in sync? And if there is, it just means you need to decide whether it's a good idea since you will need to keep it updated.

Reply:
> I think such a workaround implies that your master is your fork is not running that dashboard as well?

Correct, yes. Further comments are coming as regular comments that would likely not be hidden.